### ISSUE TITLE
Import from Scrivener

### ISSUE THREAD

**Issue by kevboh** [obsidianmd/obsidian-importer#30]
I'd like to suggest support for importing from [Scrivener](https://www.literatureandlatte.com/scrivener/overview). It's been a while since I've written my own scripts to parser their bundled format, but I know that it's doable.

**Comment by ReblochonMasque** [comment]
Scrivener has built in export to markdown hasn't it?

**Comment by OogieM** [comment]
Yes but you get 2 files, the text as a .md file and the metadata as a .txt file
